web/README.md

```diff
@@ -0,0 +1,418 @@
+# HarbourBridge: Web APIs (Experimental)
+
```

Comment:
Let's format this to be consistent with the other README.md files i.e. keep lines to around 80 characters where that makes sense.

Reply:
updated line formatting and Spanner wherever applicable